test/test_images/kafka_performance/dev.config

```diff
@@ -0,0 +1,55 @@
+### Creating this benchmark:
+# mako create_benchmark test/performance/broker-latency/dev.config
```

Comment:
This is the wrong Mako config, but we can fix that later once we start running this test in Mako.